src/main.c

```diff
@@ -91,7 +91,7 @@ static void IOTA_main()
             }
             CATCH(EXCEPTION_IO_RESET)
             {
-                THROW(EXCEPTION_IO_RESET);
+                break;
```

Comment:
Don't throw exceptions inside catch, also there as no need

Reply:
I think this one is to force a complete reset of the app for this specific exception. The same behavior is done in app-boilerplate: https://github.com/LedgerHQ/app-boilerplate/blob/master/src/main.c#L87
Shouldn't we keep it?